<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HybridTestTab</class>
 <widget class="QWidget" name="HybridTestTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="loHybridTestTab">
     <item>
      <widget class="QGroupBox" name="gbSettings">
       <property name="title">
        <string>Hybrid Test</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="lblVcth">
            <property name="text">
             <string>Vcth</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDial" name="dialVcth">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="notchesVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QDial" name="dialEvents">
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="txtVcth">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>120</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4" alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="txtEvents">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="lblEvents">
            <property name="text">
             <string>Number of Events</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="loOptionsExtra">
          <item>
           <widget class="QCheckBox" name="checkTestReg">
            <property name="text">
             <string>Test Registers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkScan">
            <property name="text">
             <string>Scan Threshold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkHoleMode">
            <property name="text">
             <string>Hole Mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnStart">
          <property name="text">
           <string>Start Hybrid Test</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerSettings">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
